recipes/scala/README.md

```diff
@@ -2,14 +2,221 @@
+`https://github.com/adobe/experience-platform-dsw-reference/blob/master/datasets/retail/DSWRetailSales.xdm.json`
+
+
+        curl -X POST \
+             https://platform.adobe.io/data/foundation/catalog/xdms/_customer/DSWRetailSales \
+             -H 'Authorization: Bearer {token} \ - Get the token from cookies of the UI 
+             -H 'Content-Type: application/json' \
+             -H 'cache-control: no-cache' \
+             -H 'x-api-key: acp_machineLearning_customer' \
+             -H 'x-gw-ims-org-id: 20656D0F5B9975B20A495E23@AdobeOrg' \
+             -d '{
+                   "title": "DSWRetailSales",
+                   "description": "string",
+                   "type": "object",
+                   "properties": {
+                     "date": {
+                       "id": "date",
+                       "type": "string"
+                     },
+                     "store": {
+                       "id": "store",
+                       "type": "integer"
+                     },
+                     "storeType": {
+                       "id": "storeType",
+                       "type": "string"
+                     },
+                     "weeklySales": {
+                       "id": "weeklySales",
+                       "type": "number"
+                     },
+                     "storeSize": {
+                       "id": "storeSize",
+                       "type": "integer"
+                     },
+                     "temperature": {
+                       "id": "temperature",
+                       "type": "number"
+                     },
+                     "regionalFuelPrice": {
+                       "id": "regionalFuelPrice",
+                       "type": "number"
+                     },
+                     "markdown": {
+                       "id": "markdown",
+                       "type": "number"
+                     },
+                     "cpi": {
+                       "id": "cpi",
+                       "type": "number"
+                     },
+                     "unemployment": {
+                       "id": "unemployment",
+                       "type": "number"
+                     },
+                     "isHoliday": {
+                       "id": "isHoliday",
+                       "type": "boolean"
+                     }
+                   }
+             }
+### Upload Dataset from UI             
+
+Upload the parquet file from here:
+`https://github.com/adobe/experience-platform-dsw-reference/blob/master/datasets/retail/DSWRetailSales.parquet`
+
+Create a dataset with "DSWRetailSales" schema and the parquet file as the source. Get this datasetID and plugin it in pipelineservice.json.
+Please look at the `Video to create dataset and upload parquet file` to platform UI.
+
+### Video to create dataset and upload parquet file
+
+[![Watch the video](../../docs/images/HomePage.png)](https://youtu.be/pRyN-Xb2cyo)
```

Comment:
the video is unavailable. Is it private?

Reply:
Yes, sorry made both the videos public.